### PR TITLE
tests: Rename tests with the naming system

### PR DESCRIPTION
**Pull request by spencer-lunarg**
Just renaming the test names to new format

### CODE REVIEW

File: tests/negative/shader_push_constants.cpp [hunk 1]
```diff
@@ -152,7 +152,7 @@ TEST_F(NegativeShaderPushConstants, NotInLayout) {
     m_errorMonitor->VerifyFound();
 }
 
-TEST_F(VkLayerTest, InvalidCmdPushConstantRange) {
+TEST_F(NegativeShaderPushConstants, CommandBufferRange) {
```
Comment (artem-lunarg) [inline tests/negative/shader_push_constants.cpp:155]: `ComamndBufferRange` a bit misleading.

**Review by artem-lunarg — APPROVED** [review]

File: tests/negative/shader_push_constants.cpp (hunk 1 above)
Comment (spencer-lunarg) [reply tests/negative/shader_push_constants.cpp:155]: reverted

Comment (spencer-lunarg) [reply tests/negative/shader_push_constants.cpp:155]: not reverted, but took suggestion of `Range`

File: tests/positive/android_hardware_buffer.cpp [hunk 2]
```diff
@@ -18,8 +18,10 @@
 #include "../framework/android_hardware_buffer.h"
 #endif
 
+class PositivAndroidHardwareBuffer : public VkPositiveLayerTest {};
```
Comment (juan-lunarg) [inline tests/positive/android_hardware_buffer.cpp:21]:
```suggestion
class PositiveAndroidHardwareBuffer : public VkPositiveLayerTest {};
```

**Review by juan-lunarg — APPROVED** [review]
1 typo

**Review by ncesario-lunarg — APPROVED** [review]

**Pull request merged**